packages/rest/src/rest.application.ts

```diff
@@ -83,6 +84,16 @@ export class RestApplication extends Application implements HttpServerLike {
     this.restServer.handler(handlerFn);
   }
 
+  /**
+   * Mount static assets to the REST server
+   * @param path The path to serve the static asset
+   * @param root The root directory
+   * @param options Options for serve-static
+   */
+  static(path: string, root: string, options?: ServeStaticOptions) {
```

Comment:
I find the names `path` and `root` too similar; it's too easy to confuse which one is for the URL path and which one is for the filesystem path. I am proposing the following renames:
 - `path` -> `url`, `urlPath` or `endpointPath`
 - `root` -> `rootDir`, `fileRoot` or `serveFrom`

Please update `RestServer#static` in the same way too.